examples/contrib/oed/eig_estimation_benchmarking.py

```diff
@@ -521,7 +609,7 @@ def main(case_tags, estimator_tags, num_runs, num_parallel, experiment_name):
                             param_func, param_params = value
                             # Communicate some size attributes to the guide
                             if "guide" in key or "classifier" in key:
-                                param_params.update({"d": expanded_design.shape[:2],
+                                param_params.update({"d": expanded_design.shape[:-2],
```

Comment:
is this correct?

Reply:
depends on the assumptions being made: typically design tensors are `(num_parallel, num_designs, observation_dim, parameter_dim)`. We could add another batch dimension, it's rare we'd add another intrinsic dimension. So I prefer `-2`